src/Core/src/Platform/iOS/SwitchExtensions.cs

```diff
@@ -10,24 +10,31 @@ public static class SwitchExtensions
 		public static void UpdateIsOn(this UISwitch uiSwitch, ISwitch view)
 		{
 			uiSwitch.SetState(view.IsOn, true);
+			uiSwitch.UpdateTrackColor(view);
```

Comment:
I think the way we want to trigger this is from the Handler not from the extension method. 

For example, like this

![image](https://user-images.githubusercontent.com/5375137/205168763-57ec1719-e170-42e7-98e0-55c9faea6ad1.png)

The reason being, if someone were to add a `mapper` for the `TrackColor` property than we'd want any requests to update that property to pass through the mapper.

Reply:
@mattleibow thoughts on my comments?

@tj-devel709 indicated that you had him move it into the extension method to keep handlers thin.  I think for cases like this the handler is what contains the logic flow and the extensions themselves shouldn't have side effects. 

Plus, if the `UpdateIsOn` extension method calls `UpdateTrackColor` that will bypass the user if the user has setup their own mapper for `MapTrackColor`

Reply:
I think I meant don't call the handler.UpdateValue but rather call the extension method - or copy the line that we need.

Otherwise, changing the value and then updates the handler and then back into a mapper. It jumps all over the place. Rather go from handler, then to extension and stay there.

Not sure what side effects there are with calling the mappers from a mapper and may result in weirdness. Unless you think that is OK?

Reply:
> How have we done other things where one property affects others?

I don't know if we've been consistent but typically any time the value of X means Y needs to be re-evaluated then I will trigger the update of Y through the mapper. 

Because let's say someone does this

```C#
SwitchHandler.Mapper.ModifyMapper("TrackColor", <my_cide>)
```

If we call `MapTrackColor` from `MapIsOn` that now bypasses the `ModifyMapper` code that the user wants running every time the `TrackColor` is changed. 

We have this a few places in the code.

![image](https://user-images.githubusercontent.com/5375137/206520316-2040f6ac-1e59-49d6-bfad-e43614f3b28f.png)

![image](https://user-images.githubusercontent.com/5375137/206520371-ced5b9dc-b25d-4a87-88eb-1b9cc0b02eb6.png)

![image](https://user-images.githubusercontent.com/5375137/206520418-36ce53b4-0a57-4c88-a37b-bf85f6ae1afe.png)

And then cases where we want to refire mappers from OnLoaded

![image](https://user-images.githubusercontent.com/5375137/206520561-ec09b205-ccad-4210-8761-1c86dd9ee378.png)

Reply:
Yeah, I suppose you convinced me :)

Reply:
Sorry @tj-devel709 seems Shane is the most powerful Ween today. Also, not sure if the order matters @PureWeen? Should we have some sort of UpdateValue-Before-UpdateControl policy?

Reply:
@mattleibow @PureWeen it looks like from the examples above, updateValue comes first so I followed that order here!